radix-engine-tests/tests/reference.rs

```diff
@@ -457,3 +404,61 @@ fn test_add_local_ref_to_stored_substate() {
         _ => false,
     });
 }
+
+fn flash_package() -> (DefaultTestRunner, PackageAddress) {
```

Comment:
Must flash the package since we've now added checks for transient blueprints in the package package.